Dockerfile

```diff
@@ -1,19 +1,11 @@
-ARG PHP_VERSION=8.1
-
-FROM php:${PHP_VERSION}-cli
+FROM php:8.1-cli
```

Comment:
Minimum PHP version supported according to `composer.json` file.

Reply:
We run the tests with the minimum supported PHP version locally. We can see the result of running tests with several versions of PHP and MongoDB when we push the changes. That's why we use strategies in the workflow.

I believe we should leave it to workflows because developers shouldn't run tests with several versions of PHP in combination with several MongoDB versions. It takes time while workflows can do it very accurately and fast.